### PR TITLE
Button

### PR DESCRIPTION
**Pull request by anshtripathi01**
**### Please Review my button component**
Button

### CODE REVIEW

File: Components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    text-decoration: none;
+    padding: 8px 10px;
+    border-radius: 2px;
+    color: antiquewhite;
+    margin-left: 10px;
+}
+
+.btn-outline-primary{
+    border: solid 1px tomato;
+    color: tomato;
```
Comment (zabihshaik) [inline Components/Button/button.css:30]: Better to use HEX values for colors instead of names, and use variables

**Review by zabihshaik — COMMENTED** [review]
Everything looks good.

File: Components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    font-size: 1.4rem;
+    font-weight: 400;
+}
+.float-btn-primary{
+    box-shadow: 0px 1px 3px 0px rgba(0, 0 ,0,0.1);
+    width: 25px;
+    height: 25px;
+    border-radius: 50%;
+    padding: 10px 12px 12px;
+    background-color: tomato;
```
Comment (shashikantmaurya0007) [inline Components/Button/button.css:58]: you can create variable dedicated to button it will be easy to maintain in future

**Review by shashikantmaurya0007 — APPROVED** [review]
Those are the few chnages you need to take care else rest looks good

**Pull request merged**